mobile/src/board/hooks/useBoard.ts

```diff
@@ -17,30 +17,73 @@ import {
   type UserAction,
 } from "./useBoardTypes";
 
+import { useGetPlaceTileHintMutation } from "./useGetPlaceTileHintMutation";
+import { useGetMovePawnHintMutation } from "./useGetMovePawnHintMutation";
+
 export function useBoard(id: number | string): [Board | null, Error | null] {
   const [board, setBoard] = useState<Board | null>(null);
   const [error, setError] = useState<Error | null>(null);
 
+  const placeTileHint = useGetPlaceTileHintMutation();
+  const movePawnHint = useGetMovePawnHintMutation();
+
   const fetchBoard = async function () {
     setError(null);
     try {
       const updatedBoard = await boardRepository.getById(id);
       setBoard(updatedBoard);
+
+      setTimeout(() => handleBotTurn(updatedBoard), 0);
     } catch (e) {
       setError({ message: "Failed to load board", severity: "error" });
       setBoard(null);
     }
   };
 
+  const handleBotTurn = async (board: Board) => {
+    if (
+      board.gameState == GameState.End ||
+      board.gameState == GameState.Animating
+    ) {
+      return;
+    }
+
+    if (!board.currentPlayer?.isBot || !board.isHumanPlayer) {
```

Comment:
I don't get the difference between `board.currentPlayer.isBot` and `board.isHumanPlayer`
I mean, `isHumanPlayer` is the same that  `!board.currentPlayer.isBot` ?